joblib/test/test_my_exceptions.py

```diff
@@ -2,6 +2,7 @@
 Test my automatically generate exceptions
 """
 from joblib import my_exceptions
+from _pytest.assertion.util import BuiltinAssertionError
```

Comment:
OK this happens for Python 2.7 fair enough. To be honest, I would say just make the test simpler by just using another kind of exception for example ImportError.